docs/pages/management/admin/uninstall-teleport.mdx

```diff
@@ -408,10 +404,7 @@ $ docker stop teleport
   ```code
   $ del C:\Path\To\tsh.exe
   ```
-
-  You can uninstall Teleport Connect from the "Apps and Features" section of the Control Panel.
-
-  For reference, Teleport Connect binaries are installed to `%LOCALAPPDATA%\Programs\teleport-connect`.
+(!docs/pages/includes/teleport-connect-telemetry.mdx!)
```

Comment:
Telemetry? 
I think you meant `(!docs/pages/includes/uninstall-teleport-connect-windows.mdx!)`

Reply:
Thanks, fixed!